org.firo.firo-qt.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.firo.firo-qt</id>
+  <name>Firo</name>
+  <metadata_license>MIT</metadata_license>
```

Comment:
Missing project license https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#license

Reply:
added